sycl/test-e2e/lit.cfg.py

```diff
@@ -574,7 +574,7 @@ def open_check_file(file_name):
     )
     config.substitutions.append(("%include_option", "/FI"))
     config.substitutions.append(("%debug_option", "/Zi /DEBUG"))
-    config.substitutions.append(("%cxx_std_option", "/std:"))
+    config.substitutions.append(("%cxx_std_option", "/Qstd:"))
```

Comment:
Use of `/Qstd` is not available in intel/llvm as it is xmain specific
```suggestion
    config.substitutions.append(("%cxx_std_option", "/clang:-std="))
```
This should work better, allows Linux specific options that work with `clang` or `clang++` to be passed along for `clang-cl`.